README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://travis-ci.org/openworklabs/quasar.svg?branch=primary)](https://travis-ci.org/openworklabs/quasar)
 
+NOTE: make sure `node -v` or `nvm ls` returns a node version of 12 or greater<br />
```

Comment:
Lol @radio-alice i meant for you to put this on thomas' other PR for documentation. taking this out.

Reply:
whoops yeah I was a little confused